js/src/controls/WidgetControl.js

```diff
@@ -0,0 +1,83 @@
+var widgets = require('@jupyter-widgets/base');
+var _ = require('underscore');
+var L = require('../leaflet.js');
+var control = require('./Control.js');
+var LeafletControlView = control.LeafletControlView;
+var LeafletControlModel = control.LeafletControlModel;
+
+L.Control.WidgetControl = L.Control.extend({
+    options: {
+        position: 'topright',
```

Comment:
The default value is 'topleft' at other places, maybe change this one to 'topleft' too?